run.py

```diff
@@ -1,5 +1,24 @@
 # Run a test server.
+import sys
+from werkzeug.serving import WSGIRequestHandler
 from annotationengine import create_app
-app = create_app()
+import os
+
+HOME = os.path.expanduser("~")
+
 if __name__ == "__main__":
-    app.run(host='0.0.0.0', port=7777, debug=True)
+    if len(sys.argv) == 2 and sys.argv[1] == "--seg":
+        seg = True
+    else:
+        seg = False
+
+    app = create_app(seg=seg)
```

Comment:
I don't see where this boolean goes... isn't it suppose to be a config dictionary?